xlive/H2MOD/Modules/Shell/Config.cpp

```diff
@@ -48,8 +48,8 @@ bool H2Config_mouse_uniform = false;
 float H2Config_controller_sens = 0;
 bool H2Config_controller_modern = false;
 H2Config_Deadzone_Type H2Config_Controller_Deadzone = H2Config_Deadzone_Type::Axial;
-float H2Config_Deadzone_A_X = 1.0f;
-float H2Config_Deadzone_A_Y = 1.0f;
+float H2Config_Deadzone_A_X = 26.0f;
```

Comment:
any reason for modifying this?

Reply:
The default deadzone ingame is 26, for some reason it's set to 1 in the config so I changed it to match that